src/main/scala-2.13-/scala/concurrent/stm/compat/package.scala

```diff
@@ -0,0 +1,18 @@
+package scala.concurrent.stm
+
+import scala.collection.mutable
+
+package object compat {
```

Comment:
Alternatively, we could add a dependency on `scala-collection-compat`. You should at least mention it in the PR description.

Reply:
This is small enough to warrant pasting it here instead of adding a new dependency.